packages/feedback/src/core/sendFeedback.ts

```diff
@@ -48,18 +51,39 @@ export async function sendFeedbackRequest(
       event: baseEvent,
     });
 
-    if (!feedbackEvent) {
-      return resolvedSyncPromise({});
+    if (client.emit) {
+      client.emit('beforeSendFeedback', feedbackEvent, { includeReplay: Boolean(includeReplay) });
     }
 
-    client.emit('beforeSendFeedback', feedbackEvent, { includeReplay: Boolean(includeReplay) });
+    try {
+      const response = await transport.send(
+        createEventEnvelope(feedbackEvent, dsn, client.getOptions()._metadata, client.getOptions().tunnel),
+      );
+
+      if (attachments && attachments.length) {
+        // TODO: https://docs.sentry.io/platforms/javascript/enriching-events/attachments/
+        await transport.send(
```

Comment:
See comment above, instead we can do `client.sendEvent(feedbackEvent, { attachments })` to send another event with the attachments.

Reply:
with either the transport.send or the client.sendEvent i'm not seeing a screenshot coming into sentry today. :(

Reply:
yea `sendEvent` currently doesn't work with attachments due to our backend. If there's an attachment and a feedback within the same envelope, the screenshot doesn't come in. We have to send them in seperate envelopes, but once the backend is fixed we will be using sendEvent `client.sendEvent(feedbackEvent, { attachments })`